### PR TITLE
[REF] .travis.yml: Enable runbot

### PR DESCRIPTION
**Pull request by moylop260**
Related with https://github.com/OCA/maintainer-quality-tools/issues/503

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: .travis.yml
Comment (lasley) [inline .travis.yml:27]: Isn't `LINT_CHECK=0` default?

**Review by lasley — APPROVED** [review]

File: .travis.yml
Comment (moylop260) [reply .travis.yml:27]: Yes, Maybe the first author is based on our template:

https://github.com/OCA/maintainer-quality-tools/blob/aafbddb5aed3dd1155609b1d16f982d832941c44/sample_files/.travis.yml#L34

Do you recommend us removed it?

Comment (pedrobaeza) [reply .travis.yml:27]: Well, explicit better than implicit, isn't it?

Comment (lasley) [reply .travis.yml:27]: Totally agree, I was more curious - not requesting change

**Comment by pedrobaeza** [comment]
Travis was red before and runbot has warnings, so I merge.

**Pull request merged**
